pom.xml

```diff
@@ -89,9 +89,15 @@
     <module>dubbo-dependencies</module>
     <module>dubbo-metadata</module>
     <module>dubbo-metrics</module>
-    <module>dubbo-spring-boot</module>
     <module>dubbo-test</module>
     <module>dubbo-maven-plugin</module>
+    <module>dubbo-spring-boot-project/dubbo-spring-boot-actuator</module>
+    <module>dubbo-spring-boot-project/dubbo-spring-boot-autoconfigure</module>
+    <module>dubbo-spring-boot-project/dubbo-spring-boot-compatible</module>
+    <module>dubbo-spring-boot-project/dubbo-spring-boot-3-autoconfigure</module>
```

Comment:
Should move to profile `jdk-version-ge-17`

Reply:
Have already existed.